launch/app/pom.xml

```
@@ -6,7 +6,7 @@
 
   <groupId>org.openhab.demo</groupId>
   <artifactId>org.openhab.demo.app</artifactId>
-  <version>2.5.5-SNAPSHOT</version>
+  <version>2.5.6-SNAPSHOT</version>
```

Since there's a new release again: 

```suggestion
  <version>2.5.7-SNAPSHOT</version>
```